src/__tests__/createAction-test.js

```diff
@@ -93,11 +93,6 @@ describe('createAction()', () => {
         type
       });
 
-      const explictNullAction = createAction(type)(null);
-      expect(explictNullAction).to.deep.equal({
-        type
-      });
-
```

Comment:
Yes, it was failing before, but I've removed it because of [your comment](https://github.com/acdlite/redux-actions/pull/128#r78083557) 😅